test/modules/string/lowerCase_test.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flodash/flodash.dart' as flodash;
+
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  // lowerCase is converts string as space separated words, to lower case.
```

Comment:
- Let's add this comment to the actual function definition instead of the test with one more / for it to serve as a documentation

- Also lets use the same wording as in the official lodash documentation `Converts string, as space separated words, to lower case.`

Reply:
Done